dbt/include/bigquery/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,18 @@
+{% macro bigquery__get_columns_spec_ddl() %}
+  {# loop through user_provided_columns to create DDL with data types and constraints #}
+  {% if config.get('constraints_enabled', False) %}
```

Comment:
Move this if statement to `adapters.sql` similar to this: https://github.com/dbt-labs/dbt-core/pull/6271/files#diff-7246f19f4d12042d172ac61b3d0c4564ea27dd8825870b35425ede1ea41625dbR28

Reply:
Looks like the if statement been [added to adapters.sql](https://github.com/dbt-labs/dbt-bigquery/pull/426/files#diff-42d3b42807f9c80bb3fe0c068d6e29df79cb404677a919fa9f6e8b5db312077aR55), and can now be deleted from this macro.

Reply:
Good catch!